__tests__/serverSpec.jsx

```diff
@@ -82,7 +82,7 @@ describe('post /', () => {
   test('responds with a 201 to a post request', () => {
     supertest(app)
       .post('/')
-      .field(35, 'my awesome avatar')
+      .field('chinese', '205')
```

Comment:
This test still would pass with the old value here. Is this because it was asynchronous and jest ended the test beforehand?